docs/maintenance/release-and-sync.md

```
@@ -92,6 +92,29 @@ bun scripts/sync-official-docs.ts --upstream ../aidlc-workflows --upstream-sha "
+
+```bash
+bun run build:artifact-map          # 再生成
+bun scripts/build-artifact-map.ts --check   # 差分があれば exit 1
+```
+
+- **結合キーはステージ番号**（`2.7`）です。見出し文言はロケールで変わりますが番号は変わりません。どの成果物が存在するかは `stage-graph.json` の `produces[]` が決め、スナップショットにしか無い行は捨てます。
+- **正規名とファイル名は一致しないことがあります**（`build-test-results` → `test-results.md`）。対応はステージファイルの `outputs:` フロントマター行から解決します。
+- **ja の construction / operation ページは成果物名そのものを翻訳しています**（`cd-config.md` → 「CD 設定文書」）。そのためファイル名では結合できず、**行の並び順**で結合します。これが成り立つ根拠は、ja がファイル名を書いている ideation / inception の全 65 行で並び順が en と一致していることで、`tests/artifact-map.test.ts` がこれを毎回測ります。ja が並べ替えたらゲートが落ちます。
+- ただしこの計測は、行順結合を実際に使うステージ（construction / operation）そのものには届きません。ja にファイル名が無いため照合できる行がゼロだからです。そこで各ステージに、ペアリングに使った **en の行順**（`joinOrder`）を記録しています。en が並べ替わって ja が追随していない場合、生成器は**書き込みを拒否**します（`--check` も exit 1）。
+
+  ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)